### PR TITLE
chore: dual-license

### LINKED ISSUE #2: Choose an OSI-approved license
Can this be offered under a standard OSI-approved license so that it is easier to consume it in an enterprise environment, and just to serve as a good example?

### PR DESCRIPTION
**Pull request by vobu**
apache 2.0 + beerware

fixes #2

### CODE REVIEW

**Review by wridgeu — APPROVED** [review]
lgtm!

File: LICENSE [hunk 1]
```diff
@@ -2,8 +2,8 @@
  * ----------------------------------------------------------------------------
  * "THE DERIVED BEER-WARE LICENSE" (Revision 1):
  * You can do whatever you want with this stuff. When you like it, just buy
- * Volker Buzek (@vobu) a beer or buy Peter Muessig (@pmuessig) a coke when
- * you see one of them.
+ * Volker Buzek (@vobu), Marco Beier (@wridgeu) or Matthias Oßwald (@matthiaso)
```
Comment (wridgeu) [inline LICENSE:5]: Actually I think Osswald is written with double s - https://people.sap.com/matthias.osswald

**Review by wridgeu — CHANGES_REQUESTED** [review]
Sorry, checked it again. Might want to change Oßwald to Osswald, and add the 'Apache-2.0' to the package.json file instead of the Beerware (as it is done in https://github.com/petermuessig/ui5-ecosystem-showcase/blob/master/package.json) other than that it looks fine.

File: LICENSE (hunk 1 above)
Comment (RandomByte) [reply LICENSE:5]: @matz3's GitHub says Oßwald 😉

SAP replicates employee names everywhere. No chance to get non-ASCII chars in there (at least not 10 years ago) 🙈

**Pull request merged**